simulator/src/test/java/com/hedera/block/simulator/generator/BlockAsFileLargeDataSetsTest.java

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright (C) 2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hedera.block.simulator.generator;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import com.hedera.block.simulator.config.data.BlockStreamConfig;
+import com.hedera.block.simulator.config.types.GenerationMode;
+import com.hedera.hapi.block.stream.BlockItem;
+import com.hedera.pbj.runtime.ParseException;
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Paths;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class BlockAsFileLargeDataSetsTest {
+
+    private final String rootFolder = "src/test/resources/block-0.0.3-blk/";
+    private int filesInFolder;
+
+    @BeforeEach
+    void setUp() {
+        filesInFolder = getFilesInFolder(getAbsoluteFolder(rootFolder));
```

Comment:
Could this be `@BeforeAll`? Does the value actually change?

Reply:
✅  Done

Reply:
Noted, resolved.